test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -45,12 +45,26 @@ class test_ucp_mmap : public ucp_test {
         return GetParam().variant;
     }
 
+    bool is_tl_rdma() {
+        /* if we have both shared memory and rdma options, it's possible that
+         * only shared memory is actually used
+         */
+        return (has_transport("dc_x") || has_transport("rc_x") ||
+                has_transport("rc_v") || has_transport("ib")) &&
```

Comment:
1. what about `ud_v` and `ud_x` TLs?
what do you think about adding `has_ib()` wrapper in `ucp_test` and then use it in `is_tl_rdma()`?
we have it in UCT tests:
```
bool uct_test::has_ib() const {
    ...
}
```

2. was it intentionally not added to the check, since it doesn't have HW RMA support? add a comment to describe it?

Reply:
is_rdma() returns whether transport has rdma capabilities (ud doesn't have), will add comment

Reply:
@yosefe thank you, I see